pkg/machines/components/vmnetworktab.jsx

```diff
@@ -69,7 +69,17 @@ class VmNetworkTab extends React.Component {
         };
 
         if (!vm.interfaces || vm.interfaces.length === 0) {
```

Comment:
This logic should not be here rather than where we call ListingTable.
ListingTable has the emptyCaption property that we should display when the Table does not have any rows.
Please use that instead of doing this here.